api/v1alpha1/istio_types.go

```diff
@@ -37,9 +37,9 @@ const (
 type IstioSpec struct {
 	// +sail:version
 	// Defines the version of Istio to install.
-	// Must be one of: v1.24.1, v1.23.3, v1.23.0, v1.22.6, v1.22.4.
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,order=1,displayName="Istio Version",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:fieldGroup:General", "urn:alm:descriptor:com.tectonic.ui:select:v1.24.1", "urn:alm:descriptor:com.tectonic.ui:select:v1.23.3", "urn:alm:descriptor:com.tectonic.ui:select:v1.23.0", "urn:alm:descriptor:com.tectonic.ui:select:v1.22.6", "urn:alm:descriptor:com.tectonic.ui:select:v1.22.4"}
-	// +kubebuilder:validation:Enum=v1.24.1;v1.23.3;v1.23.0;v1.22.6;v1.22.4
+	// Must be one of: v1.24.1, v1.23.0.
```

Comment:
we don't support the latest 1.23.x?

Reply:
It is a bit confusing, but otherwise we'd have to release two Istio versions at the same time. I think for TP this is okay. In a GA release we'd have released 1.23.3 as z-stream to the previous minor

Reply:
Currently we can only build one istio version downstream. So now we will build 1.24.1 and use already existing images for 1.23.0 which were built for TP1.